platforms/documentation/docs/src/samples/templates/problems-api-usage/reporters/standard-plugin/src/main/java/reporters/StandardPlugin.java

```diff
@@ -33,6 +33,10 @@ public void apply(Project project) {
                 .documentedAt("https://github.com/gradle/gradle/README.md")
                 .severity(Severity.WARNING)
                 .solution("Please use a more recent plugin version")
+            .additionalData(SomeAdditionalData.class, additionalData -> {
+                additionalData.setName("Some name");
+                additionalData.setNames(java.util.Arrays.asList("name1", "name2"));
```

Comment:
❔ Can we use a prettier API to pass a list?

Reply:
🤔 any suggestions?